CONTRIBUTING.md

```diff
@@ -22,7 +22,7 @@ cd rsconnect-python
 python3 -m venv .venv
 # Activate the virtual environment
 source .venv/bin/activate
-# install our requirements into the virtual environment
+# install our requirements into the virtual environment based 
```

Comment:
nit: seems like this might have been incompletely edited

Reply:
updated